.circleci/config.yml

```diff
@@ -135,14 +112,17 @@ jobs:
     steps:
       - checkout
       - install-sdkman
+      - run: sdkmanager "platforms;android-32"
+      - run: sdkmanager "build-tools;32.0.0"
       - android/accept-licenses
       - android/create-avd:
           avd-name: myavd
-          system-image: system-images;android-29;default;x86
+          system-image: system-images;android-31;google_apis;x86_64
           install: true
       - android/start-emulator:
           avd-name: myavd
           no-window: true
+          wait-for-emulator: false
```

Comment:
Maybe we want this?

Reply:
It never completes 😿 
It just waits and waits and waits. It seems to be waiting for the logs to calm down and they never do. You can seen an example of this https://app.circleci.com/pipelines/github/RevenueCat/cordova-plugin-purchases/718/workflows/ae145a1b-0168-4293-b348-6dfb09d12621/jobs/1193